lib/components/proposals/ProposalCreationForm.jsx

```diff
@@ -576,9 +568,11 @@ const getProposeParamsFromForm = (formData, t) => {
       }
 
       const contractInterface = new ethers.utils.Interface(action.contract.abi)
-      const fn = contractInterface.functions[action.contract.fn.name]
+      const fnFragment = contractInterface.fragments.find(
+        (fragment) => fragment.name === action.contract.fn.name
```

Comment:
Will this know how to discern between two methods such as `setMinter()` vs. `setMinter(address)`? Sometimes methods have the same name but different signatures

Reply:
Hmm, no I don't think so. I'm not sure how ethers handles that when it builds the fragments. From what I can tell it would just return the first match on name.

Looks like we'd need the signature to accurately handle that case. But we do this so we can get the signature...